docs/java/release-notes.mdx

```diff
@@ -27,9 +27,48 @@ We highly recommend regularly updating to the latest SDK version. It will help y
 - update client libraries giving access to latest SAP services on SAP Cloud Platform and S4/HANA
 - protect yourself from bugs and breaking changes in the future
 
+## 3.31.0 - October 22, 2020
+
+Javadoc | [Release blog](https://blogs.sap.com/?p=1207517) | [Maven Central](https://search.maven.org/search?q=g:com.sap.cloud.sdk*%20AND%20v:3.31.0)
+
+### New Functionality
+
+- Generated OData V4 client library service types provide a new method `batch()` to conveniently create instances of `BatchRequestBuilder`. It can be used to combine multiple reading and modifying OData operations into one OData Batch request.
+- We released new pre-generated type-safe client libraries (also known as Virtual Data Model - VDM) for the [latest RTC release 2020 of SAP S/4HANA On-premise](https://blogs.sap.com/2020/10/07/sap-s-4hana-2020-release/). With the exception of the following services, everything else available in [SAP API Business Hub for SAP S/4HANA On-Premise 2020](https://api.sap.com/package/S4HANAOPAPI?section=Artifacts) are available:
+    - [Settlement Document](https://api.sap.com/api/OP_API_SETTLMT_DOC_0001/resource)
+    - [Settlement Process Type - Read](https://api.sap.com/api/OP_API_SETTLMT_PROC_TYPE_0001/resource)
+    - [Responsibility Management Team Configurations - Read](https://api.sap.com/api/OP_API_RESPYM_TEAM_CONFIG_SRV_0001/resource)
+    - [Real Estate Contract (Synchronous A2X)](https://api.sap.com/api/OP_API_RECONTRACT_0001/resource)
+    - [Settlement Document Type - Read](https://api.sap.com/api/OP_API_SETTLMT_DOC_TYPE_0001/resource)
+    - [Responsibility Management Teams - Read](https://api.sap.com/api/OP_API_RESPYM_TEAM_SRV_0001/resource)
```

Comment:
```suggestion
- Generated OData V4 client library service types provide a new method `batch()` to conveniently create instances of `BatchRequestBuilder`. It can be used to combine multiple reading and modifying OData operations into one OData Batch request.
- We released new pre-generated type-safe client libraries (also known as Virtual Data Model - VDM) for the [latest RTC release 2020 of SAP S/4HANA On-premise](https://blogs.sap.com/2020/10/07/sap-s-4hana-2020-release/). Except for the following services, everything else available in [SAP API Business Hub for SAP S/4HANA On-Premise 2020](https://api.sap.com/package/S4HANAOPAPI?section=Artifacts) are available:
    - [Settlement Document](https://api.sap.com/api/OP_API_SETTLMT_DOC_0001/resource)
    - [Settlement Process Type - Read](https://api.sap.com/api/OP_API_SETTLMT_PROC_TYPE_0001/resource)
    - [Responsibility Management Team Configurations - Read](https://api.sap.com/api/OP_API_RESPYM_TEAM_CONFIG_SRV_0001/resource)
    - [Real Estate Contract (Synchronous A2X)](https://api.sap.com/api/OP_API_RECONTRACT_0001/resource)
    - [Settlement Document Type - Read](https://api.sap.com/api/OP_API_SETTLMT_DOC_TYPE_0001/resource)
    - [Responsibility Management Teams - Read](https://api.sap.com/api/OP_API_RESPYM_TEAM_SRV_0001/resource)
```